cumulus_library/template_sql/utils.py

```diff
@@ -91,21 +102,35 @@ def is_codeable_concept_array_populated(
     :param cursor: a PEP-249 compliant database cursor
     :param coding_element: the place inside the code element to look for coding info.
         default: 'coding' (and :hopefully: this is always right)
+    :allow_partial: If true, codings which do not have fields expected by the library
+        will still be included, and will need to be manually coerced.
     :returns: a boolean indicating if valid data is present.
     """
+
+    # if the source column is missing for some reason (i.e. we're dealing with
+    # conversion to FHIR rather than a true FHIR source and it's incomplete),
+    # we'll return false
     try:
         query = get_is_table_not_empty_query(table, base_col)
         cursor.execute(query)
         if cursor.fetchone() is None:
             return False
-    except:
-        return False
 
-    query = get_column_datatype_query(schema, table, base_col)
-    cursor.execute(query)
-    if coding_element not in str(cursor.fetchone()[0]):
-        return False
+        query = get_column_datatype_query(schema, table, base_col)
```

Comment:
nit: This whole method feels like a lot of tricky duplicated logic with the above.